data/containers/autoyast_containers.xml.ep

```diff
@@ -80,7 +80,7 @@
     <keep_install_network config:type="boolean">true</keep_install_network>
     % }
   </networking>
-  % if ($check_var->('VERSION', '15-SP4') || $check_var->('VERSION', '15-SP5')) {
+  % if ($get_var->('VERSION') =~ /15-SP[4,5,6]$/) {
```

Comment:
It's the first time I see commas inside square brackets in a regular expression. Shouldn't it be:

```suggestion
  % if ($get_var->('VERSION') =~ /15-SP[456]$/) {
```

Reply:
Yes! I have used `{}` at the beginning and then noticed that it might not be a great idea. Thanks

Reply:
fixed